src/components/AddDependencySearchResult/AddDependencySearchResult.js

```diff
@@ -127,20 +128,54 @@ class AddDependencySearchResult extends PureComponent<Props> {
 
     const downloadNumColor = getColorForDownloadNumber(hit.downloadsLast30Days);
 
+    // We have to use a custom highlight
+    // to unescape the html
+    const CustomHighlight = connectHighlight(
+      ({ highlight, attribute, hit, highlightProperty }) => {
+        const highlights = highlight({
+          highlightProperty: '_highlightResult',
+          attribute,
+          hit,
+        });
+
+        return highlights.map((part, i) => {
+          // Run the parser on each of the parts
+          const unescaped = (new DOMParser().parseFromString(
+            `<span>${part.value}</span>`,
+            'text/html'
+          ): any).body.textContent;
+
+          // If the part is highlighted, wrap in <mark> for the highlight
+          // Otherwise just render the part in a <span>
+          return part.isHighlighted ? (
+            <mark key={i} className="ais-Highlight-highlighted">
+              {unescaped}
+            </mark>
+          ) : (
+            <span key={i}>{unescaped}</span>
+          );
+        });
+      }
+    );
```

Comment:
I think we should move this to its own file, and make it clearer that it's a component.

I'd create a `CustomHighlight` component in `src/components`, and restructure it like:

```js
type Props = any; // If we really wanted to we could add types for this, but it's 3rd-party so it doesn't feel critical to me. But let's still create a Props type, for consistency.

const CustomHighlight = ({ highlight, attribute, hit, highlightProperty }: Props) => {
  ... All the stuff you have here
}

export default connectHighlight(CustomHighlight);
```